packages/widgets/src/blocks/legacy-widget/edit/preview.js

```diff
@@ -7,57 +7,75 @@ import classnames from 'classnames';
  * WordPress dependencies
  */
 import { useRefEffect } from '@wordpress/compose';
-import { addQueryArgs } from '@wordpress/url';
 import { useState } from '@wordpress/element';
 import { Placeholder, Spinner, Disabled } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
+import apiFetch from '@wordpress/api-fetch';
+
+async function renderPreview( idBase, instance ) {
```

Comment:
Nit: I'd probably go for `getPreviewHTML` or something like that but it's just my preference.

Reply:
Fixed in 3ea1587d917b7155a477762df9f820298b5676dc